### PR TITLE
Provide build instructions without Android studio

### PR DESCRIPTION
**Pull request by mjec**
These worked for me on 2016-03-29 on Arch Linux ;)

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@mjec, thanks for the pull request! It looks like you haven't yet signed the Dropbox CLA. Please [sign it here](https://opensource.dropbox.com/cla/).

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot**
				
	@mjec, thanks for signing the CLA!

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot**
				
	@mjec, thanks for signing the CLA!

File: README.rst [hunk 1]
```diff
@@ -34,6 +34,63 @@ emulator. Here are instructions for creating an Android virtual device
...
+
+1. Install the Android SDK including at least the API 23 (Android 6.0),
+   Build Tools, API Platform, Google APIs, Google Play Services,
+   Android Support Library, the Local Maven Repository for Support and
+   the Google Repository.
+   
+   All of these can be installed, together with their dependencies,
+   using the Android SDK manager.
+
+2. Comment out or remove references to Craslytics. The following are
```
Comment (timabbott) [inline README.rst]: typo spelling Crashlytics.

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot**
				
	@mjec, thanks for signing the CLA!

File: README.rst (hunk 1 above)
Comment (niftynei) [inline README.rst]: FYI: PR #33 removes crashlytics for debug builds.

**Comment by niftynei** [comment]
these instructions look good to me :+1:

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot**
				
	@mjec, thanks for signing the CLA!

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot**
				
	@mjec, thanks for signing the CLA!

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot**
				
	@mjec, thanks for signing the CLA!

**Comment by lfaraone** [comment]
Lgtm, will merge. Also, feel free to submit a PR ripping out Crashlytics from the open-source project if having it around causes problems — we can keep it in the build we release to the app store if we decide to continue using it.

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot**
				
	@mjec, thanks for signing the CLA!

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot**
				
	@mjec, thanks for signing the CLA!

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot**
				
	@mjec, thanks for signing the CLA!

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot**
				
	@mjec, thanks for signing the CLA!

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*
				
	@mjec, thanks for signing the CLA!

**Comment by smarx** [comment]
BTW, @mjec, sorry for the spam by the CLA bot... I made a change that caused repeated messages. It shouldn't happen again!

**Comment by mjec** [comment]
No worries @smarx, I just assumed you were really really grateful ;)

**Comment by timabbott** [comment]
Merged, thanks @mjec ! :)

**Pull request closed without merging**